mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -114,6 +118,23 @@ func parseKubectlShortVersion(version string) ([3]int, error) {
 
 // IsMesheryRunning checks if the meshery server containers are up and running
 func IsMesheryRunning(currPlatform string) (bool, error) {
+	// Get viper instance used for context to extract the endpoint from config file
+	mctlCfg, _ := config.GetMesheryCtl(viper.GetViper())
+
+	currCtx, _ := mctlCfg.GetCurrentContext()
+
+	urlEndpoint := currCtx.GetEndpoint()
+
+	urlTest := urlEndpoint + "/api/server/version"
```

Comment:
```suggestion
	urlTest := urlEndpoint + "/api/system/version"
```
Switch it to system, we are switching the URL in another PR.

Reply:
Oh ok then! 👍

Reply:
It's resolved